unchained-commons/src/main/java/unchained/commons/utils/ReflectionUtils.java

```diff
@@ -1,12 +1,14 @@
 package unchained.commons.utils;
 
+import sun.misc.Unsafe;
```

Comment:
We shouldn't be using internal APIs that are prone to deprecation and removal. (I actually think that after Java 10 these will become inaccessible with Jigsaw's deployment).